pages/events/index.vue

```diff
@@ -132,6 +135,10 @@ export default class extends QiskitPage {
   routeName: string = 'events'
   windowWidth: Number = 0
 
+  get hasEvents (): boolean {
+    return this.$store.getters.filteredEvents.length !== 0
```

Comment:
You don't need the `$store.getters` part because you already mapped it into the object in [L109](https://github.com/Qiskit/qiskit.org/pull/514/files#diff-f5bccdcfec2d036fb031c17293a33db1R109).